### ISSUE TITLE
Don't nest stuck arrows in zombie corpses on death.

### ISSUE THREAD

**Issue by Nerezza** [CleverRaven/Cataclysm-DDA#54924]
### Is your feature request related to a problem? Please describe.

One of the spectacular features of archery is that you can recover your fired arrows. Just moments before typing this, I've cleared out a few residential blocks in a suburban area using just 40 arrows.

Generally when an arrow lands a good hit but doesn't finish off a zombie, the arrow gets stuck in their body. This adds it to their drops on death, so most of the time you're looting your arrows back from zombie corpses.

Now that zombie corpses are nesting their inventories on death though, it's impossible to find stray arrows using V to list nearby items, and autopickup does not look inside corpses for items which means you have to manually go through streets full of zombie corpses trying to locate arrows you've lost.

### Solution you would like.

For the sake of not making archery a huge pain in the butt, I suggest having arrows drop outside corpses when a creature dies. Visually the player would be able to spot their arrows, and when they're having trouble finding the proverbial (or is it literal) needle in a haystack of 100 zombie corpses, then can just press V and retrieve.

### Describe alternatives you have considered.

1) Writing this up as a bug report because autopickup doesn't reach into corpses to grab your arrows. Would still be really difficult to find which corpses you need to walk by to collect all your stray arrows.

2) Do the autopickup-reaches-in-corpses thing and on top of that make V show everything nested inside corpses. Not sure why it doesn't already but I'm assuming there's a reason for this, and technically nyxing fired arrows from nesting is a smaller gameplay change than this solution.

3) A visual indicator on corpses that have arrows stuck in them. This would be a totally new game feature which would be an overcomplicated fix for a little problem imho.

**Comment by LeahLuong** [comment]
What version of the game are you running?? Did you notice #54406 when you ran a search before posting so as not to create a duplicate issue??

**Comment by Nerezza** [comment]
Literally just updated the game again so I don't know the version I was running, but my last update to it was only a couple days ago.

I did not notice that PR, I looked for anyone making a suggestion for un-nesting arrows specifically. However, the problem with "fixing" this by having auto-pickup reach into corpses is that the player would still have to walk up to literally every single corpse whereas before nested corpses became a thing you just had to press V and walk between all the visible arrows.

Basically, the point of this request is different from the point of that request.
